geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/api/MembershipBuilder.java

```diff
@@ -32,6 +32,8 @@
 
   MembershipBuilder<ID> setMembershipListener(MembershipListener<ID> membershipListener);
 
+  MembershipBuilder<ID> setMembershipLocator(MembershipLocator<ID> membershipLocator);
+
```

Comment:
this method on the builder is the replacement for the eliminated `started()` method on `LifecycleListener`